### PR TITLE
Setup GitHub pipeline

### PR DESCRIPTION
**Pull request by lucas-puerari**
https://makeitapp.atlassian.net/browse/MCRD-1171

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,4 +4,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
\ No newline at end of file
+## [Unreleased]
+
+### Added
+
+[MCRD-1171](https://makeitapp.atlassian.net/browse/MCRD-1171): Added github pipeline
```
Comment (gbarbon) [inline CHANGELOG.md:11]: @lucas-puerari let's remove the code and PR link

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -16,8 +16,8 @@ before contributing. If you find someone that is not respecting it please report
 There are several ways to contribute. You can [fork the repository](#fork) and [propose a Pull Request](CONTRIBUTING.md#how-to-submit-a-pr), or you can start a discussion to report a bug, an indesired behavior of the application, or to simply suggest a feature.
 
 Before opening a new thread, please verify whether there's already a discussion about the topic of your interest. You can find them:
-- in the [issues tab](https://github.com/mia-platform/crud-service/issues);
-- in the [official Mia Platform Community Discussion page](https://github.com/mia-platform/community/discussions?discussions_q=label%3A%22CRUD+Service%22);
+- in the [issues tab](https://github.com/mia-platform/ocr-service/issues);
```
Comment (gbarbon) [inline CONTRIBUTING.md:19]: Not 100% sure, but it should be correct like this:

```suggestion
- in the [issues tab](https://github.com/mia-care/ocr-service/issues);
```

File: default.env [hunk 3]
```diff
@@ -1,2 +1 @@
-LOG_LEVEL=DEBUG
-HEADER_KEYS_TO_PROXY=miauserid,miausergroups,miaclienttype,client-type,x-request-id
```
Comment (gbarbon) [inline default.env:2]: @lucas-puerari we don't need any headers to be proxied?

**Review by gbarbon — CHANGES_REQUESTED** [review]

File: default.env (hunk 3 above)
Comment (lucas-puerari) [reply default.env:2]: Nope, currently the ocr service is an end point service

**Review by gbarbon — APPROVED** [review]

**Pull request merged**